src/nodes/display/RenderOutputNode.js

```diff
@@ -40,7 +40,7 @@ class RenderOutputNode extends TempNode {
 
 		// output color space
 
-		if ( outputColorSpace === SRGBColorSpace ) {
+		if ( outputColorSpace !== LinearSRGBColorSpace ) {
```

Comment:
probably should be working color space

Reply:
If `outputColorSpace` is a valid color space and not equal to the working color space, then you must do the conversion.

Reply:
makes sense, I will implement support closer to `ColorManagement.workingColorSpace`, until now I was avoiding using it.